pywinauto/findwindows.py

```diff
@@ -255,12 +257,14 @@ def _title_match(w):
         elements = [elem for elem in elements if elem.enabled]
 
     if best_match is not None:
+        # Build a list of wrapped controls.
+        # Speed up the loop by setting up local pointers
         wrapped_elems = []
+        add_to_wrp_elems = wrapped_elems.append
+        wrp_cls = backend_obj.generic_wrapper_class

```

Comment:
Is it really running faster? I thought module attribute resolution wouldn't be a bottleneck.